src/plugins/intel_gna/src/runtime/gna_float_runtime.cpp

```diff
@@ -71,7 +71,12 @@ void FP::infer() {
             break;
         }
         case kDnnMaxPoolOp: {
-            ApplyMaxPoolTransform(comp, kDnnFloat);
+            bool is_fused_with_convolution_2d = false;
```

Comment:
Again, the right fix should be fusing maxPool with Convolution in the transformation (Convolution->GNAConvolution)